keyboards/lily58/lib/layer_state_reader.c

```diff
@@ -6,8 +6,8 @@
 #define L_BASE 0
 #define L_LOWER 2
 #define L_RAISE 4
-#define L_ADJUST 8
-#define L_ADJUST_TRI 14
+#define L_ADJUST 65536
```

Comment:
```suggestion
#define L_ADJUST (1 << 16)
```

Reply:
As a note, non-consecutive layer numbering introduces pointless empty layers between them, inflating the firmware size for no good reason, but that may be out of scope here.